common/transport/amqp/src/amqp.ts

```diff
@@ -75,14 +30,13 @@ class PutTokenStatus {
  * @param   {String}    sdkVersionString        String identifying the SDK used (device or service).
```

Comment:
>(device or service) [](start = 80, length = 19)

confusing.  this implies that the value could be "device" or "service", but I think it's more qualified than that.